experiment/measurer.py

```diff
@@ -67,6 +67,10 @@ def remote_dir_exists(directory: pathlib.Path) -> bool:
 
 def measure_loop(experiment: str, max_total_time: int):
```

Comment:
How many measures will be run in parallel when running locally?

Reply:
As many as there are cores on the machine. This will interfere with experiments so we probably need to add something to limit this later on.